XenModel/XenAPI/FriendlyErrorNames.resx

```diff
@@ -1676,6 +1682,9 @@ Authorized Roles: {1}</value>
   <data name="VIF_IN_USE" xml:space="preserve">
     <value>Network has active VIFs</value>
   </data>
+  <data name="VIF_NOT_IN_MAP" xml:space="preserve">
+    <value>This VIF was not mapped to a destination Network in VM.migrate_send operation</value>

```

Comment:
"to a destination network</value>" (lower case "network", and end the string there).